### PR TITLE
Specs brands and suppliers

### PR DESCRIPTION
**Pull request by Saimis777**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Please be specific when describing the PR. What did you add, why did you submit it?
| Fixed ticket? | Fixes #{issue number here} if there is a related issue

### CODE REVIEW

File: content/1.7/back-office/catalog/brands-and-suppliers/add-edit-brand.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+
+8) Enabled
+
+This is a toggle switch that can be turned to "Yes" or "No", by default it is set to "Yes"
+
+9) Shop association
+
+The shop association block is displayed when the multistore is enabled and if several shops are configured.
+
+👉 See the [specs of the shop association block]({{<ref "../../../multistoregeneralspecs.md#shop-association-block">}} "Shop association block")
```
Comment (fabienvallon) [inline content/1.7/back-office/catalog/brands-and-suppliers/add-edit-brand.md:182]: => should be ../../multistoregeneralspecs.md#shop-association-block

File: content/1.7/back-office/catalog/brands-and-suppliers/brands-and-suppliers-brands.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+7. Country
+
+Drop-down list of all the countries which can be selected and searched.
+
+8. Actions
+
+**Edit.** The brand can be edited as often as necessary.
+
+👉 See the [specs of the add/edit address]({{<ref "add-edit-address.md">}} "Add/edit address")
```
Comment (fabienvallon) [inline content/1.7/back-office/catalog/brands-and-suppliers/brands-and-suppliers-brands.md]: [add/edit address]({{<ref "add-edit-address.md">}} "Add/edit address")

File: content/1.7/back-office/catalog/brands-and-suppliers/brands-and-suppliers-brands.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
+---
+title: Brands and suppliers brands
+weight: 1
+---
+
+In this page Brands are addresses are displayed.
```
Comment (fabienvallon) [inline content/1.7/back-office/catalog/brands-and-suppliers/brands-and-suppliers-brands.md:6]: brands and adresses

File: content/1.7/back-office/catalog/brands-and-suppliers/brands-and-suppliers-brands.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+ 3) Add new brand address
+
+👉 See the [specs of the add/edit address]({{<ref "add-edit-address.md">}} "Add/edit address") 
+
+### Settings wheel
+
+When clicking on the settings wheel, the following options are displayed:
+1) Import
+
+After clicking on it, the Advanced parameters > Import page is opened.
```
Comment (fabienvallon) [inline content/1.7/back-office/catalog/brands-and-suppliers/brands-and-suppliers-brands.md:25]: the  [Import page]({{<ref "../../advanced-parameters/import/">}} "Import Page")  is opened.

**Comment by Saimis777** [comment]
> Desk phone ?

@fabienvallon hi could you clarify this suggestion?

**Pull request merged**